app/assets/javascripts/short-form/templates/c4-household-public-housing.html.slim

```diff
@@ -1,14 +1,22 @@
 .app-card.form-card
   short-form-header
-    h2.app-card_question
-      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE' | translate: youOrHouseholdTranslateVariable() }}
+    h2.app-card_question ng-if="householdMembers.length == 0"
+      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_YOU' | translate }}
+    h2.app-card_question ng-if="householdMembers.length > 0"
+      | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_HOUSEHOLD' | translate }}
     p.form-note.margin-bottom
       | {{'C4_HOUSEHOLD_PUBLIC_HOUSING.P1' | translate}}
 
   .app-inner.inset
     .form-group ng-class="{ error: inputInvalid('STUB_householdPublicHousing') }"
       p.form-note.margin-bottom translate="LABEL.PLEASE_SELECT_ONE"
-      .radio-group.padding-top--half role="radiogroup" aria-label="{{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE' | translate}}"
+      .radio-group.padding-top--half role="radiogroup" aria-label="{{'C4_HOUSEHOLD_PUBLIC_HOUSING.TITLE_YOU' | translate}}" ng-if="householdMembers.length == 0"
+        radio-block-item value='Yes' user='application' name='STUB_householdPublicHousing' ng-required='true' on-change="publicHousingYes"
```

Comment:
@chrisdolendo yes, right idea but not quite, the aria-label doesn't belong on the h2, the aria-label is the accessibility label for the radio group to indicate which text is labelling it. so more like

```slim
h2#question-title.app-card_question ng-if="householdMembers.length == 0"
  | {{ 'C6A_HOUSEHOLD_RESERVED_UNITS_VETERAN.TITLE_YOU' | translate }}
...
.radio-group.padding-top--half role="radiogroup" aria-labelledby="question-title"
```

and I just used `question-title` as an example ID, basically the ID of the h2 has to match with the labelledby attr.